### PR TITLE
[accel brake map calibrator] sync main / .auto porting

### PR DESCRIPTION
**Pull request by tkimura4**
## Description
use .auto msgs
package: [accel_brake_map_calibrator]

## Review Procedure
ros2 launch accel_brake_map_calibrator accel_brake_map_calibrator.launch.xml 

## Pre-Review Checklist for the PR Author

**PR Author should check the checkboxes below when creating the PR.**

- [ ] Read [pull request guidelines][pull-request-guidelines]
- [ ] Code follows [coding guidelines][coding-guidelines]
- [ ] Assign PR to reviewer

## Checklist for the PR Reviewer

**Reviewers should check the checkboxes below before approval.**

- [ ] Commits are properly organized and messages are according to the guideline
- [ ] Code follows [coding guidelines][coding-guidelines]
- [ ] (Optional) Unit tests have been written for new behavior
- [ ] PR title describes the changes

## Post-Review Checklist for the PR Author

**PR Author should check the checkboxes below before merging.**

- [ ] All open points are addressed and tracked via issues or tickets
- [ ] Write [release notes][release-notes]

## CI Checks

- **Build and test for PR / build-and-test-pr**: Required to pass before the merge.
- **Build and test for PR / clang-tidy-pr**: NOT required to pass before the merge. It is up to the reviewer(s). Found false positives? See the [guidelines][clang-tidy-guidelines].
- **Check spelling**: NOT required to pass before the merge. It is up to the reviewer(s). See [here][spell-check-dict] if you want to add some words to the spell check dictionary.

[clang-tidy-guidelines]: https://tier4.github.io/autoware.proj/tree/main/developer_guide/ClangTidyGuideline/
[coding-guidelines]: https://tier4.atlassian.net/wiki/spaces/AIP/pages/1194394777/T4
[pull-request-guidelines]: https://tier4.github.io/autoware.proj/tree/main/developer_guide/PullRequestGuideline/
[release-notes]: https://tier4.atlassian.net/wiki/spaces/AIP/pages/563774416
[spell-check-dict]: https://github.com/tier4/autoware-spell-check-dict#how-to-contribute

### CODE REVIEW

File: vehicle/accel_brake_map_calibrator/accel_brake_map_calibrator/README.md [hunk 1]
```diff
@@ -0,0 +1,132 @@
+# Accel Brake Map Calibrator
```
Comment (taikitanaka3) [inline vehicle/accel_brake_map_calibrator/accel_brake_map_calibrator/README.md:1]:
```suggestion
# accel_brake_map_calibrator
```

File: vehicle/accel_brake_map_calibrator/accel_brake_map_calibrator/launch/accel_brake_map_calibrator.launch.xml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  <arg name="logger_level" default="info" />
+  <arg name="rviz" default="true" />
+  <arg name="pedal_accel_graph_output" default="true" />
+  <arg name="progress_file_output" default="false" />
+  <arg name="use_sim_time" default="false" />
+
+  <!-- calibration interface -->
+  <node pkg="accel_brake_map_calibrator" exec="accel_brake_map_calibrator" name="accel_brake_map_calibrator" output="screen">
+    <param from="$(var accel_brake_map_calib_param)"/>
+    <remap from="~/input/odometry" to="/vehicle/status/velocity_status" />
```
Comment (taikitanaka3) [inline vehicle/accel_brake_map_calibrator/accel_brake_map_calibrator/launch/accel_brake_map_calibrator.launch.xml:15]:
```suggestion
    <remap from="~/input/velocity_status" to="/vehicle/status/velocity_status" />
```

Comment (tkimura4) [reply vehicle/accel_brake_map_calibrator/accel_brake_map_calibrator/launch/accel_brake_map_calibrator.launch.xml]: Done 6ae9f19

File: vehicle/accel_brake_map_calibrator/accel_brake_map_calibrator/README.md (hunk 1 above)
Comment (tkimura4) [reply vehicle/accel_brake_map_calibrator/accel_brake_map_calibrator/README.md]: ccbc6e9

File: vehicle/accel_brake_map_calibrator/accel_brake_map_calibrator/src/accel_brake_map_calibrator_node.cpp [hunk 3]
```diff
@@ -0,0 +1,1413 @@
...
+    RCLCPP_WARN(
+      rclcpp::get_logger("accel_brake_map_calibrator"), "Cannot read brakemap. csv path = %s. ",
+      output_brake_file_.c_str());
+  }
+}
+
+void AccelBrakeMapCalibrator::callbackVelocity(
+  const autoware_auto_vehicle_msgs::msg::VelocityReport::ConstSharedPtr msg)
+{
+  // convert odometry to twiststamped
```
Comment (taikitanaka3) [inline vehicle/accel_brake_map_calibrator/accel_brake_map_calibrator/src/accel_brake_map_calibrator_node.cpp:447]:
```suggestion
  // convert odometry to twist stamped
```

**Review by taikitanaka3 — APPROVED** [review]

**Comment by taikitanaka3** [comment]
```
/accel_brake_map_calibrator/debug/data_average_occ_map
/accel_brake_map_calibrator/debug/data_count_occ_map
/accel_brake_map_calibrator/debug/data_count_self_pose_occ_map
/accel_brake_map_calibrator/debug/data_count_self_pose_occ_map_updates
/accel_brake_map_calibrator/debug/data_std_dev_occ_map
/accel_brake_map_calibrator/debug/occ_index
/accel_brake_map_calibrator/debug/original_occ_map
/accel_brake_map_calibrator/debug/original_raw_map
/accel_brake_map_calibrator/debug/update_occ_map
/accel_brake_map_calibrator/output/current_map_error
/accel_brake_map_calibrator/output/debug_values
/accel_brake_map_calibrator/output/map_error_ratio
/accel_brake_map_calibrator/output/update_raw_map
/accel_brake_map_calibrator/output/update_suggest
/accel_brake_map_calibrator/output/updated_map_error
/checkpoint
/diagnostics
/initialpose
/move_base_simple/goal
/parameter_events
/planning/mission_planning/route
/rosout
/sensing/gnss/pose_with_covariance
/simulation/dummy_perception_publisher/object_info
/tf
/tf_static
/vehicle/status/actuation_status
/vehicle/status/steering_status
/vehicle/status/velocity_status
```

File: vehicle/accel_brake_map_calibrator/accel_brake_map_calibrator/src/accel_brake_map_calibrator_node.cpp (hunk 3 above)
Comment (tkimura4) [reply vehicle/accel_brake_map_calibrator/accel_brake_map_calibrator/src/accel_brake_map_calibrator_node.cpp]: 32ab478

(odometryの方もおかしいので修正)

**Pull request merged**